.github/workflows/build-and-test-snap.yml

```
@@ -31,8 +64,9 @@ jobs:
         env:
           MOCK_GPIO: true
           SKIP_TEARDOWN_REMOVAL: true
-          LOCAL_SERVICE_SNAP: ../${{ steps.snapcraft.outputs.snap }}
-        run: go test -failfast -p 1 -timeout 20m -v
+          LOCAL_SERVICE_SNAP: ../${{ needs.build-arm64.outputs.snap }}
```

You're referring it using the environment variable which gives you an absolute path, removing `../` should fix it: 

```suggestion
          LOCAL_SERVICE_SNAP: ${{ needs.build-arm64.outputs.snap }}
```